setup.go

```diff
@@ -94,6 +95,22 @@ func setup(c *caddy.Controller) error {
 			if len(args) == 1 {
 				fp.probeResistDomain = args[0]
 			}
+		case "serve_pac":
```

Comment:
Are there other things this plugin could do with a PAC file? If this is all there is, we could name this just "pac" which is simpler. Or maybe "pacfile". I dunno.

Also - tangent: It might be worth renaming "experimental_probe_resist" to just "probe_resistant" or "hidden" or something, and use the docs to clearly indicate it is experimental. That way there won't be breaking changes as the feature matures.